Plugins: Update plugin signing copy + docs (#34716)
grafana/grafana: docs/sources/plugins/catalog.md

# Plugin catalog

> The Plugin catalog allows users to browse and manage plugins from within Grafana. Only Grafana Server Admins can access and use the catalog.

Reviewer: ```suggestion
The plugin catalog allows you to browse and manage plugins from within Grafana. Only a Grafana server administrator can access and use the catalog.
```
